liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/ButtonDTO.kt

```diff
@@ -446,43 +441,20 @@ internal class ButtonDTO private constructor(builder: Builder) :
         }
 
         /**
-         * The border width to draw around the container of this button. This property is used just
-         * for `OutlineButton`.
-         * ```
-         * <OutlinedButton border-width="2">...</OutlinedButton>
-         * ```
-         * @param borderWidth int value representing button border's width.
-         * content.
-         */
-        fun borderWidth(borderWidth: String): Builder = apply {
-            if (borderWidth.isNotEmptyAndIsDigitsOnly()) {
-                this.borderWidth = (borderWidth.toIntOrNull() ?: 0).dp
-            }
-        }
-
-        /**
-         * The border color to draw around the container of this button. This property is used just
-         * for `OutlineButton`.
+         * The border to draw around the container of this button. This property is used just for
+         * `OutlineButton`.
          * ```
-         * <OutlinedButton border-color="#FF0000FF">...</OutlinedButton>
+         * <OutlinedButton border="{'width': '2', 'color': '#FF0000FF'}">...</OutlinedButton>
          * ```
-         * @param borderColor int value representing the padding to be applied to the button's
-         * content. The color must be specified as a string in the AARRGGBB format.
+         * @param border a JSON representing the border object. The `width` key is an int value
+         * representing button border's width content. The `color` key must be specified as a string
+         * in the AARRGGBB format
```

Comment:
```suggestion
         * @param border JSON representing the border object. The `width` key is an int value
         * representing the button border's width content. The `color` key must be specified as a string
         * in the AARRGGBB format
```

Would the `color` key also support the `system-*` colors? If so, this comment and the other could be updated to indicate that.

Reply:
Yes, it is supporting `system-*` colors. Could you please open a task for that? I need to update the docs in almost all components so that it would add a lot of non-related code in this PR. 
Also, could you suggest a text to document the system colors?

Reply:
Issue [#192](https://github.com/liveview-native/liveview-client-jetpack/issues/192) opened.